conf/modules/fdm_gazebo.xml

```diff
@@ -0,0 +1,92 @@
+			    &lt;spherical_coordinates&gt;
+					&lt;surface_model&gt;EARTH_WGS84&lt;/surface_model&gt;
+					&lt;latitude_deg&gt;51.9906&lt;/latitude_deg&gt;
+					&lt;longitude_deg&gt;4.37679&lt;/longitude_deg&gt;
+					&lt;elevation&gt;0&lt;/elevation&gt;
+					&lt;heading_deg&gt;180&lt;/heading_deg&gt;&lt;!-- Temporary fix for issue https://bitbucket.org/osrf/gazebo/issues/2022/default-sphericalcoordinates-frame-should --&gt;
+				&lt;/spherical_coordinates&gt;
+      3. Prepare the airframe file (see examples/ardrone2_gazebo.xml):
+         a) Select Gazebo as the FDM (Flight Dynamics Model)
+              &lt;target name=&quot;nps&quot; board=&quot;pc&quot;&gt;
+      			&lt;module name=&quot;fdm&quot; type=&quot;gazebo&quot;/&gt;
+    		  &lt;/target&gt;
+		 b) Add actuator thrusts and torques to the SIMULATOR section:
+		      &lt;section name=&quot;SIMULATOR&quot; prefix=&quot;NPS_&quot;&gt;
+		        &lt;define name=&quot;ACTUATOR_NAMES&quot; value=&quot;nw_motor, ne_motor, se_motor, sw_motor&quot; type=&quot;string[]&quot;/&gt;
+		        &lt;define name=&quot;ACTUATOR_THRUSTS&quot; value=&quot;1.55, 1.55, 1.55, 1.55&quot; type=&quot;double[]&quot;/&gt;
+		        &lt;define name=&quot;ACTUATOR_TORQUES&quot; value=&quot;0.155, -0.155, 0.155, -0.155&quot; type=&quot;double[]&quot;/&gt;
+		        ...
+		      &lt;section&gt;
+		    The thrusts and torques are expressed in SI units (N, Nm) and should
+		    be in the same order as the ACTUATOR_NAMES.
+		 c) In the same section, bypass the AHRS and INS as these are not
+		    supported yet:
+		      &lt;section name=&quot;SIMULATOR&quot; prefix=&quot;NPS_&quot;&gt;
+		        ...
+		        &lt;define name=&quot;BYPASS_AHRS&quot; value=&quot;1&quot;/&gt;
+    			&lt;define name=&quot;BYPASS_INS&quot; value=&quot;1&quot;/&gt;
+		        ...
+		      &lt;section&gt;
+		 d) If required, enable video thread simulation:
+		      &lt;section name=&quot;SIMULATOR&quot; prefix=&quot;NPS_&quot;&gt;
+		        ...
+		        &lt;define name=&quot;SIMULATE_VIDEO&quot; value=&quot;1&quot;/&gt;
+		        ...
+		      &lt;section&gt;
+		 e) If required, specify the Gazebo world and aircraft name:
+		      &lt;section name=&quot;SIMULATOR&quot; prefix=&quot;NPS_&quot;&gt;
+		        ...
+		        &lt;define name=&quot;GAZEBO_WORLD&quot; value=&quot;my_world.world&quot;/&gt;
+		        &lt;define name=&quot;GAZEBO_AC_NAME&quot; value=&quot;my_uav&quot;/&gt;
+		      &lt;section&gt;
+	  4. Make sure all included modules work with nps. At the moment, most of
+	     the modules that depend on video_thread are only built when ap is
+	     selected as the target. As a quick fix, try to remove the target
+	     attribute from the makefile element in the module xml, e.g.:
+	       &lt;makefile target=&quot;ap&quot;&gt; ---&gt; &lt;makefile&gt; 
+    </description>
+  </doc>
+  <header/>
+  <makefile target="nps">
+  	<raw>
+  	nps.CXXFLAGS  += $(shell pkg-config gazebo --cflags)
```

Comment:
Gazebo's flags include `-std=c++11`. While this flag can be added to CFLAGS, it will cause compilation warnings for the other C files as the C++11 flag does not make sense there. I added CXXFLAGS to make sure that this flag is only used during C++ compilation (i.e. nps_fdm_gazebo.cpp).